docs/data/charts/export/ExportChartAsImage.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import Stack from '@mui/material/Stack';
+import Button from '@mui/material/Button';
+import { ChartProApi } from '@mui/x-charts-pro/ChartContainerPro';
+import FormControl from '@mui/material/FormControl';
+import FormLabel from '@mui/material/FormLabel';
+import RadioGroup from '@mui/material/RadioGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Radio from '@mui/material/Radio';
+import TextField from '@mui/material/TextField';
+import { LineChartPro } from '@mui/x-charts-pro/LineChartPro';
+
+function ExportParamsSelector({
+  apiRef,
+}: {
+  apiRef: React.RefObject<ChartProApi | undefined>;
+}) {
+  const [type, setType] = React.useState('image/png');
+  const [rawQuality, setRawQuality] = React.useState('0.9');
+  const quality = Math.max(0, Math.min(1, Number.parseFloat(rawQuality)));
```

Comment:
A slider woudl solve the need for such a validation :)

Reply:
Isn't a slider a bit harder to use when selecting a specific value? For floating point numbers we would have a step of 0.01, which wouldn't be great when selecting a number, right? 

I suppose sliders work better for integers.